### ISSUE TITLE
Update Uzbek translations

### ISSUE THREAD

**Issue by anorprogrammer** [squidfunk/mkdocs-material#5022]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "uz",
  "action.edit": "Ushbu sahifani tahrirlash",
  "action.skip": "Tarkibga o'tish",
  "action.view": "Ushbu sahifaning manbasini ko'rish ⬅️",
  "announce.dismiss": "Buni boshqa ko'rsatma ⬅️",
  "blog.archive": "Arxiv ⬅️",
  "blog.categories": "Kategoriyalar ⬅️",
  "blog.categories.in": "ichida ⬅️",
  "blog.continue": "O'qishni davom ettiring ⬅️",
  "blog.draft": "Qoralama ⬅️",
  "blog.index": "Indeks sahifasiga qaytish ⬅️",
  "blog.meta": "Metadata ⬅️",
  "blog.references": "Bog'liq havolalar ⬅️",
  "clipboard.copy": "Buferga nusxalash",
  "clipboard.copied": "Buferga nusxalandi",
  "consent.accept": "Qabul qilish ⬅️",
  "consent.manage": "Sozlamalarni boshqarish ⬅️",
  "consent.reject": "Rad etish ⬅️",
  "footer": "Pastgi qism",
  "footer.next": "Keyingi sahifa",
  "footer.previous": "Oldingi sahifa",
  "header": "Sarlavha",
  "meta.comments": "Izohlar",
  "meta.source": "Manba",
  "nav": "Navigatsiya",
  "readtime.one": "1 daqiqa o'qish ⬅️",
  "readtime.other": "# daqiqa o'qish ⬅️",
  "rss.created": "RSS tasmasi ⬅️",
  "rss.updated": "Yangilangan kontentning RSS tasmasi ⬅️",
  "search": "Qidirish",
  "search.placeholder": "Qidirish",
  "search.share": "Ulashish",
  "search.reset": "Tozalash",
  "search.result.initializer": "Qidiruv ishga tushirilmoqda",
  "search.result.placeholder": "Qidiruvni boshlash uchun kiriting",
  "search.result.none": "Mos natijalar yo'q",
  "search.result.one": "1 ta mos natija",
  "search.result.other": "# ta mos keladigan natijalar",
  "search.result.more.one": "Ushbu sahifada yana 1 ta natija",
  "search.result.more.other": "Bu sahifada yana # ta natija",
  "search.result.term.missing": "To'ldirilmagan",
  "select.language": "Tilni tanlang",
  "select.version": "Versiyani tanlang",
  "source": "Repozitoriyga o'tish",
  "source.file.contributors": "Contributors ⬅️",
  "source.file.date.created": "Yaratildi",
  "source.file.date.updated": "Oxirgi yangilanish",
  "tabs": "Yorliqlar",
  "toc": "Mundarija",
  "top": "Yuqoriga qaytish"
}[key] }}{% endmacro %}
```

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to my best knowledge.
- [ ] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by squidfunk** [comment]
Thanks! Added in 8aec99e9a. Two translations are still missing:

``` js
  ...
  "blog.meta": "Metadata ⬅️",
  ...
  "source.file.contributors": "Contributors ⬅️",
  ...

**Comment by anorprogrammer** [comment]
```
  ...
  "blog.meta": "Metama'lumot ⬅️",
  ...
  "source.file.contributors": "Hissa qo'shuvchilar ⬅️",
  ...
```
  Is it enough or should I open a new issue?

**Comment by squidfunk** [comment]
Thanks again! No, that's not necessary. Already added in 090ae5205.

**Comment by squidfunk** [comment]
Released as part of [9.0.13](https://github.com/squidfunk/mkdocs-material/releases/tag/9.0.13).
